reporting/impl/src/main/resources/reports/templates/problem-centric-report.ftl

```diff
@@ -101,6 +102,26 @@
                     });
                 });
             });
+            $.tablesorter.addParser({ 
+   		 id: 'thousands',
```

Comment:
parser with id thousands. We need this, because we use "," as a delimter of thousands and without this "1,355" was sorted as "1".